generators/app/templates/_dockerTaskGeneric.sh

```diff
@@ -38,8 +38,13 @@ compose () {
   else
     echo "Running compose file $composeFileName"
     docker-compose -f $composeFileName kill
-    docker-compose -f $composeFileName up -d
     
+    if [[ $ENVIRONMENT = "release" ]]; then
```

Comment:
The idea is that in debug we use volume sharing to share the source code inside of the container - we don't have to rebuild the image each time as the changes are automatically made inside of the container. For release, we'd want to re-build the image as we don't want to use the cached instance of it.